knowledge-base/grid-dynamically-adjusting-column-header-styles.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Dynamically Adjusting Column Header Cell Style in Blazor Grid
```

Comment:
This title starts with two long words before we get to the important bit. Consider something shorter like 
```suggestion
title: Adjust Grid Column Header Styles Dynamically
```